python/cudf/cudf/core/udf/utils.py

```diff
@@ -306,6 +253,8 @@ def _compile_or_get(
     we then obtain the return type from that separate compilation and
     use it to allocate an output column of the right dtype.
     """
+    # runtime check for CEC mode which is disabled for CUDA 12 for now
```

Comment:
Avoid acronyms here for clarity.
```suggestion
    # runtime check for CUDA Enhanced Compatibility mode which is disabled for CUDA 12 for now
```

Reply:
Resolving this due to removing this comment.